code/modules/antagonists/abductor/abductor.dm

```diff
@@ -152,12 +168,17 @@
 	var/static/team_count = 1
 	///List of all brainwashed victims' minds
 	var/list/datum/mind/abductees = list()
+	/// If we will recolor these aliens, this value gets changed. Has a really small chance to occur naturally, but admins can change this to anything they want.
+	var/team_skincolor = null
```

Comment:
If you want to go the extra mile for admins, you can also implement this in vv_edit_var (if that's what it's called I don't fully remember) to update bodyparts of all team members when admins change the color.